app/tests/spec/lib/crypto/recovery-keys.js

```diff
@@ -0,0 +1,179 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const {assert} = require('chai');
+const RecoveryKeys = require('lib/crypto/recovery-keys');
+
+const uid = 'aaaaabbbbbcccccdddddeeeeefffff00';
+const recoveryKey = '0123456789ABCDEFGHJKMNPQRSTVWXYZ';
```

Comment:
This recovery key doesn't actually start with the correct version number.  I wonder if `getRecoveryJwk` should reject it on that basis?